src/App.js

```diff
@@ -0,0 +1,34 @@
+import './styles/App.scss';
+import React from 'react';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import MainPage from './components/HomePage/MainPage';
+import Details from './components/DoctorsData/DetailsPage/Details';
+import Appointments from './components/AppointmentsData/AppointmentsPage';
+import LoginPage from './components/Authentication/LoginPage';
+import RegistrionPage from './components/Authentication/RegistrionPage';
+import SplashPage from './components/SplashPage/SplashPage';
+import AddItem from './components/DoctorsData/AddItem';
+import DeleteItem from './components/DoctorsData/DeleteItem';
+import AppointmentsDetails from './components/AppointmentsData/DetailsPage/Details';
+import AddAppointmentsForm from './components/AppointmentsData/AddAppointment';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

![image](https://user-images.githubusercontent.com/57421823/213647621-ea7dc7f7-acb2-4836-89d0-f54b90ce586e.png)